src/main/kotlin/no/nav/familie/ef/sak/repository/domain/MellomlagretBrev.kt

```diff
@@ -0,0 +1,10 @@
+package no.nav.familie.ef.sak.repository.domain
+
+import org.springframework.data.annotation.Id
+import java.util.UUID
+
+data class MellomlagretBrev(@Id
+                            val behandlingId: UUID,
+                            val brevverdier: String,
+                            val brevmal: String,
+                            val versjon: String)
```

Comment:
Bruker ikke denne foreløpig, men kan bli aktuelt.

Reply:
Til hva kan den eventuelt bli aktuellt?

Reply:
Hvis vi gjør "breaking changes" i sanity er det fint med en mulighet til å innvalidere mellomlagrede brev som vi vet vil krasje.

Reply:
Ja, det tror jeg er lurt. Men den settes ved lagring fra frontend, eller? Isåfall kan det gjøres med en gang tenker jeg?

Reply:
Høres lurt ut, enig med Charlie sin kommentar 👍

Reply:
Fikset i https://github.com/navikt/familie-ef-sak/pull/797/commits/0841982a824a122a91e9cdb05ae0909bc2160ccc